README.md

```diff
@@ -183,3 +183,19 @@ __Runtime network setup with CNI__
 4. Create bridge, TAP, and link all together with network interface previously created ([code](https://github.com/containers/virtcontainers/blob/0.5.0/network.go#L123-L205))
 
 5. Start VM inside the netns and start the container ([code](https://github.com/containers/virtcontainers/blob/0.5.0/api.go#L66-L70))
+
+## Storage
+
+Container workloads are shared with the virtualized environment through 9pfs.
+The devicemapper storage driver is a special case. The driver uses dedicated block devices rather than formatted filesystems, and operates at the block level rather than file level. We use this knowledge to directly use the underlying block device instead of the overlay file system for the container root file system. The block device maps to the top read-write layer for the overlay. This approach gives much better I/O performance as compared to using 9pfs to share the container file system.
+
+The approach above does introduce a limitation in terms of dynamic file copy in/out of the container via docker cp operations:
+
+```
+docker cp [OPTIONS] CONTAINER:SRC_PATH DEST_PATH|-
+docker cp [OPTIONS] SRC_PATH|- CONTAINER:DEST_PATH
+```
+
+While the copy to container from host is not expected to work, it is possible to copy files from the container to host. We do need to sync the files to the block device before the copy operation.
```

Comment:
"from host ... to container" sounds more natural to me.

Overall, I find these two sentences confusing as it is unclear to the reader if they need to do anything "special": who needs to sync the files? virtcontainers, or is it up to the user to call `sync(1)` potentially multiple times to ensure the data is persisted?